<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphsForm</class>
 <widget class="QWidget" name="GraphsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Ускорение X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ускорение Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ускорение Z</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Угловая скорость X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Угловая скорость Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Угловая скорость Z</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Магнитное поле X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Магнитное поле Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Магнитное поле Z</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Температура</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>В отдельное окно</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCustomPlot" name="plotWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>ui/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
